app/Models/Users/User.php

```diff
@@ -58,4 +58,21 @@ public function accepts()
     {
         return $this->hasMany('App\Models\Flights\Flight', 'acceptee_id');
     }
+
+    /**
+     * Required for the WebDevEtc\BlogEtc package.
+     * Enter your own logic (e.g. if ($this->id === 1) to
+     *   enable this user to be able to add/edit blog posts
+     *
+     * @return bool - true = they can edit / manage blog posts,
+     *        false = they have no access to the blog admin panel
+     */
+    public function canManageBlogEtcPosts()
+    {
+        if (\App::environment('local')) {
+            return $this->username === 'DoverEightMike';
```

Comment:
Consider using the Spatie Permissions package to check to see if the user has a certain role
```suggestion
            return $this->hasRole('admin')
```
(should work if `$this` is a user object)

Reply:
Didn't know we had that functionality already. Thanks :)